preview/Win7Msix/Win7MSIXInstaller/MsixRequest.cpp

```diff
@@ -26,30 +26,37 @@
 #include "Protocol.hpp"
 #include "FileTypeAssociation.hpp"
 #include "InstallComplete.hpp"
-
+#include "ErrorHandler.hpp"
 
 // MSIXWindows.hpp define NOMINMAX because we want to use std::min/std::max from <algorithm>
 // GdiPlus.h requires a definiton for min and max. Use std namespace *BEFORE* including it.
 using namespace std;
 #include <GdiPlus.h>
 
+struct ErrorHandlerInfo
+{
+    CreateHandler create;
+    PCWSTR handlerName;
+};
+
 struct HandlerInfo
 {
     CreateHandler create;
     PCWSTR nextHandler;
+    ErrorHandlerInfo errorHandler;

```

Comment:
>ErrorHandlerInfo [](start = 4, length = 16)

this could be simpler if this was just PCWSTR for the name, and that the errorHandler was just one of the HandlerInfo in the array instead of a separately typed ErrorHandlerInfo

Reply:
Updated structure to include an entry for error handler.